proto/lighthouse-result.proto

```diff
@@ -185,36 +127,14 @@ message LighthouseResult {
     }
 
     // The form factor used in the audit
-    EmulatedFormFactor emulated_form_factor = 8;
+    EmulatedFormFactor emulated_form_factor = 1;
```

Comment:
@paulirish should we renumber at 1, or reserve 1-16 and start at 17?  IMO we can go ahead and renumber since no one is using this yet.

Reply:
We'll need to apply two annotations as described in go/proto-best-practices-checkers but I think this is the right move for now.

LG on the renumber.